<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="UsernameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="AgeError">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;* Age is less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="AccountType">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="UserButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AdminButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="UsernameError">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;* Alredy existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="MatchingError">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;* Not matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="RegisterError">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;* All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pass2Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="Gender">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="MateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FemaleButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="BirthDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth</string>
   </property>
   <widget class="QLabel" name="MonthLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLabel" name="DayLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="YearLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QComboBox" name="MonthBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Jan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Feb</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jun</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aug</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sep</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oct</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nov</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dec</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="DayBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="YearBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2023</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2022</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2021</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2020</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2019</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2018</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2017</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2016</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2015</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2014</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2013</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2012</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2011</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2010</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2009</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2008</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2007</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2006</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2005</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2004</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2003</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2002</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1999</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1998</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1997</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1996</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Genre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Favourite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="ActionBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ComedyBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RomanceBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DramaBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HorrorBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="OtherBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="PassLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pssword</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
